docs/src/api_bindings.md

```diff
@@ -13,8 +13,8 @@ Note that in many cases, high-level data types are valid arguments through autom
 For instance, `HDF5Datatype` objects will be automatically converted to their `hid_t` ID
 by Julia's `cconvert`+`unsafe_convert` `ccall` rules.
 
-There may be additional helper wrappers (often for out-argument functions) which are
-not documented here.
+There are additional helper wrappers (often for out-argument functions) which are not
```

Comment:
Apparently I didn't update the docs after changing the corresponding sentence in the generator in the original PR :-\